### ISSUE TITLE
Which model and config file was used to train the model for AIY kit

### ISSUE THREAD

**Issue by abhinavrawat27** [dvillevald/hand_gesture_classifier#1]
Hi

Thanks for the great tutorial. I learned a basic understanding on how to retrain model for new classes. I am also trying to achieve similar functionality but unable to decide which model to train. Unfortunately, tensorflow model zoo does not contain the embedded version of the model and google has also not provided any models for AIY kit. 

It will good if you can share which model you used and what was the configuration for training the model. Thanks

**Comment by dvillevald** [comment]
Thank you for your interest! I retrained the last layer (classifier) of **mobilenet_0.50_160** model on images with resolution 160x160 pixels:

> python-m script.retrain \
--bottleneck_dir=tf_files/bottlenecks\
--how_many_training_steps=1500\
--model_dir=tf_files/models/\
--summaries_dir=tf_files/training_summaries/"mobilenet_0.50_160"\
--output_graph=tf_files/hand_gesture_classifier.pb\
--output_labels=tf_files/hand_gesture_labels.txt\
--architecture="mobilenet_0.50_160"\
--image_dir=tf_files/training_data \
--random_brightness 15

I have a [tutorial](https://www.hackster.io/dvillevald/transfer-learning-model-on-google-aiy-vision-kit) on Hackster.io which covers training step in more detail.

Let me know if it helped.

**Comment by abhinavrawat27** [comment]
@dvillevald Thanks for your transfer learning tutorial. I have few basic questions.

I have multiple objects in my training images. Can I label them using software like [labelimage](https://github.com/tzutalin/labelImg) and then pass on the xml and jpg files to the trainig process.?

In your code, you are simply detecting the hand gesture and performing action. I need to detect object and then put a bounding box around it. Is it possible ?
